WooCommerce/Classes/POS/Presentation/Settings/PointOfSaleSettingsStoreDetailView.swift

```diff
@@ -79,6 +91,45 @@ struct PointOfSaleSettingsStoreDetailView: View {
     }
 }
 
+// Temporary: Simplified copy from PointOfSaleOrderListView.GhostOrderRowView
+private struct GhostSettingRowView: View {
```

Comment:
This one is a mere copy of existing `GhostOrderRowView`, where I removed one of the extra lines. I'll iterate on this when I do the final design iteration tomorrow/next week.